Task2/index.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link href="css/content.css" rel="stylesheet" type="text/css">
+    <meta charset="UTF-8">
+    <title>Task2</title>
+</head>
+<body>
+<h1 id="b">Chapters</h1>
```

Comment:
What does the attribute `id="b"` mean? Please always give semantic names for everything: tags, classes, attributes, id.